src/check/runner/configuration/Parameters.ts

```diff
@@ -91,6 +91,19 @@ export interface Parameters<T = void> {
    * @remarks Since 1.19.0
    */
   markInterruptAsFailure?: boolean;
+  /**
+   * Skip runs with already covered cases
+   *
+   * NOTE: values comparison can be configured with 'valuesComparator'.
+   * By default values are compared by equality of fc.stringify results.
+   */
+  skipEqualValues?: boolean;
+  /**
+   * Replace the default way to compare values by a custom one
+   *
+   * Should be used with 'skipEqualValues'
+   */
+  valuesComparator?: (a: T, b: T) => boolean;
```

Comment:
Not sure of the naming 🤔 As this parameter is useless when the other is not used I'm wondering if we could expose it differently